package-overrides/github/eternicode/bootstrap-datepicker@1.3.1.json

```diff
@@ -0,0 +1,18 @@
+{
+    "main": "js/bootstrap-datepicker",
+    "shim": {
+      "js/bootstrap-datepicker": {
+        "deps": [
+          "css",
```

Comment:
This line isn't necessary - rather add `css` as a dependency of the package to download - https://github.com/jspm/registry/blob/master/package-overrides/github/twbs/bootstrap%403.1.0.json#L5. Also is Bootstrap itself a dependency?